CHANGELOG.develop

```diff
@@ -3349,6 +3349,8 @@ Other:
   - README.md:
     - Fixed macOS documentation to install fonts (thanks to Sergio Morales)
     - Updated Windows section, suggest official Emacs (thanks to ghost-420)
+  - layers/+filetree/treemacs/README.org:
```

Comment:
For pure doc changes we have a separate section in the changelog called `Various documentation improvements`. Please remove the newly added text and add your name to the list of contributors.